packages/pdf.lua

```diff
@@ -89,15 +89,33 @@ SILE.registerCommand("pdf:link", function (options, content)
   })
 end)
 
+SILE.registerCommand("pdf:metadata", function (options, content)
+  local key = SU.required(options, "key", "pdf:metadata")
+  local val = SU.required(options, "val", "pdf:metadata")
+  SILE.typesetter:pushHbox({
+    value = nil,
+    height = 0,
+    width = 0,
+    depth = 0,
+    outputYourself = function (self,typesetter)
+      SILE.outputter._init()
+      pdf.metadata(key, val)
+    end
+  })
+end)
+
 return { documentation = [[\begin{document}
 The \code{pdf} package enables (basic) support for PDF links and table-of-contents
-entries. It provides the three commands \command{\\pdf:destination}, \command{\\pdf:link}
-and \command{\\pdf:bookmark}.
+entries. It provides the four commands \command{\\pdf:destination}, \command{\\pdf:link},
+\command{\\pdf:bookmark}, and \command{\\pdf:metadata}.
 
 The \command{\\pdf:destination} parameter creates a link target; it expects a
 parameter called \code{name} to uniquely identify the target. To create a link to
 that location in the document, use \code{\\pdf:link[dest=\goodbreak{}name]\{link content\}}.
 
+To set arbitrary key-value metadata, use something like \code{\\pdf:metadata[key=Author,
```

Comment:
Is there a list of the most common/canonical values to set that we could tip people off to here?

For reference perhaps some of all of [Pandoc's meta data keys](https://pandoc.org/MANUAL.html#metadata-variables) maps to something useful and we could document those.